posix-configs/SITL/init/ekf2/standard_vtol

```diff
@@ -57,7 +57,7 @@ param set SYS_AUTOSTART 13006
 param set SYS_MC_EST_GROUP 2
 param set SYS_RESTART_TYPE 2
 param set VT_MOT_COUNT 4
-param set VT_TRANS_THR 0.75
+param set VT_F_TRANS_THR 0.75
```

Comment:
Do we need some logic to automatically update existing values from `VT_TRANS_THR` to `VT_F_TRANS_THR` for people who have set `VT_TRANS_THR`?

Reply:
@sanderux I defaulted the param to 100%